app/models/registration.rb

```diff
@@ -614,7 +620,8 @@ def attempt_auto_accept(mode)
 
   private def auto_accept_failure_reason(mode)
     return Registrations::ErrorCodes::OUTSTANDING_FEES if outstanding_entry_fees.positive?
-    return Registrations::ErrorCodes::AUTO_ACCEPT_NOT_ENABLED if auto_accept_preference_disabled? || (auto_accept_preference.to_sym != mode)
+    return Registrations::ErrorCodes::AUTO_ACCEPT_NOT_ENABLED if auto_accept_preference_disabled? ||
+                                                                 (auto_accept_preference == 'bulk' && auto_accept_preference.to_sym != mode)
```

Comment:
...doesn't this specific part of the expression (the && in brackets) just boil down to `mode != :bulk`?

Reply:
I'm not sure what that's doing there in the first place - removed and checking if any tests fail